tests/framework/extensions/charts/gatekeeperconfig.go

```diff
@@ -5,12 +5,12 @@ import "gopkg.in/yaml.v2"
 // Generates a config file for OPA gatekeeper https://open-policy-agent.github.io/gatekeeper/website/docs/exempt-namespaces#exempting-namespaces-from-gatekeeper-using-config-resource
 
 func GenerateGatekeeperConfigYaml(excludedNamespaces []string, processes []string, name string, namespace string, apiVersion string, kind string) (string, error) {
-	confMatch := ConfigMatch{
+	confMatch := configMatch{
 		{ExcludedNamespaces: excludedNamespaces,
 			Processes: processes},
 	}
 
-	confSpec := ConfigSpec{
+	confSpec := configSpec{
```

Comment:
Why did you make the structs private?

Reply:
Should they be public? I think I might've misunderstood your earlier comment about adding comments to exported structs, it didn't really feel necessary for these to be used outside of the package and I didn't want to add a bunch of clutter to the godocs.

Reply:
That makes sense, I made them public again and added comments